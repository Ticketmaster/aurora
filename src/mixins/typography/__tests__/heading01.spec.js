import HEADING_01 from "../heading01";

describe("HEADING_01", () => {
  it("should return the correct values", () => {
    expect(HEADING_01).toMatchSnapshot();
  });
});
