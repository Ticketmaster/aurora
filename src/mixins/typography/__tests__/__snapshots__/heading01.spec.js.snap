// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HEADING_01 should return the correct values 1`] = `
Array [
  "
  ",
  "
  color: ",
  [Function],
  ";
",
  ";
  ",
  "
  font-size: ",
  "20px",
  ";
",
  ";
  ",
  "
  ",
  "font-weight: 600",
  ";
",
  ";
  ",
  "
  line-height: ",
  "1.25",
  ";
",
  ";

  ",
  "
  @media screen and ",
  "(min-width: 768px)",
  " {
    ",
  "
        ",
  "
  font-size: ",
  "24px",
  ";
",
  ";
    ",
  ";
  }
",
  ";
",
]
`;
